litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml

```diff
@@ -226,47 +226,53 @@
 
           when: openebses_active.json.data == []
 
-        ## Checking the data plane label on the first node
-        - name: Checking the label on first node of the cluster
-          uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[0] }}"
-            method: GET
-            url_username: "{{ username.stdout}}"
-            url_password: "{{ password.stdout}}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            status_code: 200
-          register: node1
-          failed_when: "node1.json.nodeType['mayadata.io/data-plane'] != 'true'"
-
-        ## Checking the data plane label on the second node
-        - name: Checking the label on the second node of the cluster
-          uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[1] }}"
-            method: GET
-            url_username: "{{ username.stdout}}"
-            url_password: "{{ password.stdout}}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            status_code: 200
-          register: node2
-          failed_when: "node2.json.nodeType['mayadata.io/control-plane'] != 'true'"        
-
-        ## Checking the data plane label on the third node
-        - name: Checking the label on the third node of the cluster
-          uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[2] }}"
-            method: GET
-            url_username: "{{ username.stdout}}"
-            url_password: "{{ password.stdout}}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            status_code: 200
-          register: node3
-          failed_when: "node3.json.nodeType['mayadata.io/control-plane'] != 'true'"
+        ## Checking the data plane label on the first node                                                                                                                                                          
+        - name: Checking the label on the first node of the cluster                                                                                                                                                    
```

Comment:
Rename the task to - `Checking data plane label on the first node of the cluster`